frontend/src/pages/Home.jsx

```diff
@@ -31,34 +52,355 @@ class Home extends Component {
           </Typography>
           <Button 
             variant="contained" 
-            color="primary"
             style = {{
-              marginRight: 10
+              marginRight: 10,
+              color: "#0F1111",
+              background: "#f0c14b",
+              fontWeight: 600
+            }}
+            onClick = {()=>{
+              this.props.setLogin_dialog(true)
+              this.props.setRegister_dialog(false)
             }}
           >
-            Sign in 
-          </Button>
-          <Button variant="contained" color="primary">
-            Register
+            Log In
           </Button>
+          
+          {/* This is for the login dialog */}
```

Comment:
Maybe we can wrap this into another component file called "LoginDialog", and replace this section with <LoginDialog />, such that it's easier to maintain in the future(?

Reply:
EDIT: I just notice you have that below already lol 
~~But refactoring this may require rethinking about event handlers. For example, we may add a onCloseHandler function in this "Home" component, and pass that reference to the "LoginDialog" component.~~
<!---
```
onCloseHandler would be something like this:

const onCloseHandler = (e) => { // not sure if we need to put e as argument
   this.props.setLogin_dialog(false) // or just `setLogin_dialog(false)`
   this.props.setRegister_dialog(false) // or just `setRegister_dialog(false)`
}
```
```
<Dialog> would look something like this:

<Dialog ... onClose={this.onCloseHandler} ... />
```
--->
~~(PS: I can be wrong! ~~